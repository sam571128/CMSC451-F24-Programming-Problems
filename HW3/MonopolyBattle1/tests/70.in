36 2784 413556419
5 4
28 34
14 36
31 28
36 13
24 26
34 15
21 21
32 2
36 26
35 15
23 35
23 20
20 23
11 21
34 26
36 27
2 22
7 20
26 3
26 25
22 36
19 3
10 16
21 19
9 2
21 30
29 18
34 25
25 21
29 10
18 29
6 34
14 19
33 29
4 2
27 25
5 3
26 7
9 31
32 28
28 35
21 33
30 33
12 11
28 20
34 20
28 28
10 29
20 27
6 6
8 26
33 21
23 34
4 23
18 3
25 13
10 32
19 24
13 16
24 2
21 11
24 2
20 24
1 28
33 29
34 23
9 33
26 29
8 21
7 23
30 35
17 18
14 13
35 27
6 20
18 20
20 31
9 9
23 8
10 19
20 34
7 33
14 29
10 2
26 16
17 28
27 7
7 12
32 27
30 27
21 20
30 23
6 14
28 4
24 20
28 11
29 32
19 19
12 13
27 19
7 20
14 5
7 35
3 18
32 9
6 19
35 25
5 34
26 22
2 15
35 8
2 19
25 34
9 29
9 6
27 28
2 16
12 14
2 31
29 36
35 27
17 7
3 9
5 21
8 14
9 26
4 13
21 12
34 32
21 7
8 7
1 21
31 28
5 15
30 32
26 2
23 17
15 12
33 34
33 8
20 23
23 1
12 18
23 32
22 26
15 29
21 20
25 15
11 17
32 25
13 21
12 33
22 27
34 16
1 23
6 4
7 23
35 9
2 15
7 29
8 9
32 24
36 21
15 12
36 13
21 35
14 22
13 16
24 18
4 9
20 36
9 36
24 36
14 9
18 4
9 10
9 12
15 19
2 30
28 6
12 18
33 5
25 24
1 15
11 25
28 18
20 10
3 20
35 1
26 25
12 2
9 19
23 11
20 30
22 32
23 9
23 24
30 15
3 30
10 26
29 36
14 4
31 8
27 21
8 10
21 10
17 23
32 25
4 6
15 26
16 2
4 27
19 17
2 10
27 36
6 22
21 12
31 33
25 10
18 33
8 32
30 34
36 10
9 30
4 30
25 14
1 12
20 18
2 3
21 20
21 26
8 26
11 12
30 35
24 20
22 10
1 34
34 11
13 24
16 8
1 26
25 26
7 30
25 25
19 33
33 20
1 28
8 18
8 4
7 13
2 11
21 23
10 24
7 29
16 27
13 7
21 34
15 6
20 29
29 9
10 27
16 7
7 19
27 18
19 24
2 4
26 21
30 5
8 18
3 2
34 5
5 1
19 22
24 15
8 4
4 24
19 13
33 26
20 22
8 11
20 21
8 3
21 1
30 31
29 33
1 36
10 25
7 8
11 27
26 31
21 15
20 4
11 20
12 14
7 33
7 16
19 13
32 21
5 28
19 28
11 24
26 14
24 24
7 23
9 4
27 17
25 7
31 32
33 31
6 18
28 33
36 9
33 26
6 13
28 7
27 35
25 11
4 28
28 32
9 10
2 21
15 30
3 30
16 4
17 18
21 19
8 12
35 6
36 16
2 8
29 19
25 21
32 18
21 35
11 3
14 26
33 6
32 31
23 6
7 5
8 22
7 7
2 32
21 32
15 21
17 12
24 35
22 32
26 14
8 5
32 20
17 33
22 8
20 33
1 2
9 10
17 22
32 8
30 25
17 34
7 27
13 24
6 16
32 35
25 1
18 12
19 32
34 25
21 10
10 11
6 7
12 19
28 5
6 25
20 21
24 8
13 13
8 20
34 12
17 18
27 26
29 32
12 9
30 6
7 32
30 1
5 7
26 4
9 14
35 35
5 22
16 14
30 23
22 16
16 21
12 33
4 30
11 27
20 13
11 14
7 23
36 28
22 20
2 34
32 22
34 4
3 15
19 13
29 18
25 1
32 14
29 21
34 34
3 3
14 15
23 31
9 17
13 3
26 26
21 24
18 7
23 10
4 24
36 16
31 17
23 18
35 19
9 13
7 35
14 3
26 4
5 36
21 12
20 22
15 3
10 6
10 28
7 4
24 30
27 35
26 12
27 23
4 21
15 1
2 36
6 35
9 25
10 33
12 11
5 33
27 29
33 24
24 29
20 15
25 23
21 2
32 19
13 28
27 19
3 34
16 10
29 35
13 7
36 3
25 35
24 17
34 15
2 22
33 27
20 5
5 5
20 2
36 34
14 18
36 13
5 26
7 11
34 20
20 21
21 28
36 2
18 16
10 13
9 13
32 36
35 4
7 24
3 2
18 14
21 12
12 23
22 7
25 1
18 1
33 20
4 6
34 22
34 21
29 31
35 16
9 30
3 31
3 14
19 14
19 26
6 18
13 2
25 4
1 27
19 34
25 6
6 29
33 10
24 25
26 22
23 12
28 12
18 7
10 9
9 10
10 10
5 9
26 20
2 10
32 22
7 24
10 25
21 11
11 3
21 9
1 9
17 19
13 11
15 10
18 8
10 11
7 31
7 15
36 3
25 28
16 12
13 18
22 36
23 2
35 15
11 14
18 20
26 11
17 34
30 10
13 11
1 12
22 20
27 22
16 3
31 10
15 11
18 12
4 32
7 13
34 32
20 7
30 11
32 11
15 9
24 34
3 35
18 1
15 1
1 21
31 2
23 16
36 30
28 6
13 15
22 16
15 5
16 36
8 7
25 15
35 4
3 14
30 26
14 22
7 30
21 29
12 10
5 2
20 13
5 12
31 15
7 4
5 11
29 30
12 19
25 32
22 19
5 27
20 6
4 8
25 24
32 4
24 2
13 25
21 29
28 35
12 35
33 12
7 36
29 8
31 7
2 13
22 31
22 4
8 27
28 6
2 13
16 4
1 26
16 28
21 5
14 26
31 28
7 36
6 34
31 6
7 28
14 22
33 27
36 1
6 23
2 24
4 7
35 32
21 31
20 19
11 21
25 25
1 12
26 28
4 17
19 36
18 24
24 11
31 19
1 14
35 20
31 7
33 22
33 9
27 32
8 23
24 13
15 21
8 9
12 36
12 27
1 15
12 33
8 25
34 32
36 16
26 29
29 13
24 28
8 22
27 5
13 22
22 14
35 28
26 6
28 1
30 22
6 13
19 22
6 11
24 4
13 14
7 6
18 4
29 7
30 22
22 18
8 18
27 3
22 12
30 32
30 2
22 36
13 4
33 32
11 13
16 5
29 4
18 12
22 19
33 10
2 16
19 16
10 22
30 12
18 25
33 13
18 11
1 20
30 8
25 10
2 3
6 18
14 26
31 17
27 27
31 9
28 21
18 26
26 27
16 1
5 25
5 23
9 33
10 31
17 7
27 32
15 18
23 8
34 25
19 3
23 14
23 29
17 11
11 16
29 27
27 7
22 6
28 21
10 15
5 7
31 11
11 30
11 16
25 2
29 20
1 18
29 2
13 21
32 33
7 10
1 20
9 1
2 11
21 12
21 17
34 27
3 19
2 8
21 20
30 4
5 27
13 32
21 30
6 9
5 11
21 16
36 6
21 27
11 24
13 30
2 35
30 31
5 27
27 23
8 27
34 22
28 10
4 31
32 11
16 29
32 33
17 3
29 27
23 29
35 5
19 9
2 1
12 11
26 20
34 18
35 22
3 20
33 30
13 6
29 27
29 29
22 20
11 10
33 5
4 36
1 11
18 36
8 8
4 16
24 25
20 29
31 10
31 27
36 18
36 21
16 24
21 26
6 35
36 30
30 35
7 18
30 21
21 36
36 15
35 21
17 6
35 32
26 26
13 26
10 34
29 28
21 9
10 19
16 2
19 1
18 21
28 13
28 31
5 11
26 2
27 30
29 35
17 8
7 11
17 5
9 18
32 36
24 35
22 32
32 25
32 25
8 27
19 5
1 34
4 11
31 30
2 11
26 33
21 4
26 23
11 17
15 11
26 24
34 28
28 2
19 25
7 9
33 30
29 28
36 17
20 3
18 2
35 1
14 18
4 16
4 36
20 20
28 16
30 32
14 6
16 32
3 1
21 33
27 1
29 16
34 4
36 16
35 30
23 7
16 16
33 33
27 21
3 8
26 1
29 33
17 16
35 30
14 21
28 18
20 28
35 16
29 35
9 36
15 30
21 1
4 15
4 25
13 6
24 16
11 2
29 22
29 35
4 32
20 33
30 31
3 20
19 25
35 34
13 29
8 34
4 19
26 13
13 30
33 28
32 25
18 16
24 31
16 25
31 27
7 6
35 20
8 31
18 9
3 18
31 34
36 23
11 10
15 32
24 11
4 26
10 30
35 9
33 20
21 36
12 1
21 19
29 9
12 19
31 30
18 14
3 28
10 28
15 25
10 27
26 5
13 20
10 7
23 24
36 33
24 15
28 36
21 28
34 15
2 27
2 7
22 34
17 33
23 17
18 20
27 22
2 6
8 6
12 20
28 30
34 19
2 11
29 36
12 14
9 28
2 12
35 25
13 15
22 15
21 27
15 23
31 6
32 36
1 17
16 21
36 13
9 1
6 18
9 11
24 31
15 22
8 29
7 22
22 11
9 9
30 25
34 19
24 6
25 19
4 24
27 11
3 27
7 3
24 6
4 23
19 1
34 13
13 6
21 32
2 4
18 16
25 7
35 19
32 26
23 1
30 27
4 5
7 14
8 18
28 6
24 7
12 35
32 27
20 27
17 14
11 6
1 6
23 17
7 23
22 18
28 8
18 34
3 19
22 10
6 1
22 3
7 19
31 30
20 22
13 12
19 24
3 7
27 29
14 4
29 27
3 15
27 7
12 22
5 14
3 25
15 36
23 7
20 28
33 36
2 5
24 10
16 31
29 29
19 20
32 30
25 36
21 3
21 34
25 5
1 24
10 9
6 15
2 4
31 17
33 31
11 36
2 16
11 18
18 31
29 8
13 21
15 29
30 24
5 9
3 25
2 11
12 21
15 14
27 25
1 14
27 27
22 16
30 2
16 17
16 21
1 19
27 25
4 34
34 30
23 33
12 10
35 31
7 10
23 10
35 19
25 7
27 30
7 32
12 9
27 36
17 11
20 31
28 8
7 16
15 35
11 21
36 22
13 14
13 34
33 18
29 28
3 4
34 34
14 36
20 7
25 28
28 30
5 31
14 32
24 17
27 1
5 9
29 23
14 22
18 3
25 10
5 18
31 14
6 22
4 36
31 30
21 12
14 9
8 16
20 12
20 27
1 27
34 21
13 12
9 14
32 20
3 19
11 4
18 4
33 22
8 13
11 9
35 16
33 12
21 24
11 24
5 21
33 33
9 30
23 34
26 18
9 34
23 28
4 31
10 6
27 6
14 18
27 9
27 36
35 36
33 9
13 5
25 16
3 12
18 31
11 11
18 22
24 19
6 13
14 9
14 5
22 3
18 27
8 32
14 25
35 28
19 34
21 13
15 18
12 15
14 8
1 33
7 35
25 13
11 10
19 12
35 32
8 20
21 24
16 4
27 5
10 12
23 36
3 10
8 21
15 2
23 18
6 23
11 16
9 31
18 7
7 19
34 11
32 29
9 1
21 17
30 15
30 36
17 29
35 24
26 24
16 1
7 35
33 17
2 29
18 30
12 1
4 7
20 26
23 10
13 27
5 17
15 8
9 4
26 15
18 17
35 24
12 5
17 29
2 32
10 27
2 20
16 7
28 34
4 6
27 20
10 33
28 26
15 22
26 9
13 9
19 11
15 7
1 16
22 19
34 7
13 36
5 35
18 29
6 10
6 33
22 14
17 12
4 23
21 19
2 6
34 28
22 25
4 3
21 32
13 6
32 2
2 26
13 22
14 4
22 14
31 29
11 36
27 30
7 12
9 33
11 11
17 34
7 7
28 23
15 25
29 33
24 13
33 13
32 13
8 35
17 31
2 10
10 14
7 33
28 29
35 20
35 17
35 5
23 18
6 12
20 9
9 32
2 21
7 23
17 25
25 18
30 19
7 34
20 13
30 8
12 7
8 22
9 30
31 4
33 32
18 33
2 6
22 31
30 13
10 30
4 9
8 9
33 24
2 23
20 8
14 27
17 9
30 9
26 28
29 31
21 35
8 1
19 7
22 30
33 25
24 17
14 9
30 20
26 14
8 24
30 31
13 19
33 23
24 32
8 19
33 23
3 27
33 7
11 36
20 30
22 5
16 36
9 5
33 32
5 11
20 17
30 11
6 23
3 14
27 8
5 13
32 23
12 26
4 8
36 18
24 34
33 9
3 15
7 2
15 20
27 1
22 27
33 30
9 15
16 32
13 27
8 12
29 15
11 2
17 26
20 20
26 22
3 14
12 19
8 15
27 20
3 26
22 1
25 30
3 7
34 31
4 23
30 7
16 35
32 26
12 32
10 30
11 2
18 35
11 7
14 16
9 17
18 29
32 15
36 34
19 8
3 22
23 23
25 14
2 16
4 2
2 34
22 4
26 7
16 21
32 7
27 17
34 1
10 31
13 9
6 11
1 36
30 10
19 15
15 22
5 17
21 19
6 20
16 27
35 36
28 22
7 20
26 3
16 34
29 18
20 23
25 19
5 6
7 34
26 16
11 13
19 34
29 31
28 35
9 5
3 33
23 36
29 4
15 17
9 36
26 4
10 18
15 20
12 31
23 26
34 33
14 21
31 11
27 16
29 22
20 8
20 31
24 2
1 9
23 35
9 29
5 4
5 9
3 34
21 24
13 18
33 34
30 29
34 33
4 34
23 9
33 16
24 28
19 20
23 23
12 5
30 7
35 35
3 36
18 7
16 18
9 10
11 7
1 26
10 31
2 15
4 32
33 11
35 30
36 27
4 12
33 4
2 17
27 23
31 8
26 25
17 36
11 7
26 25
6 15
21 5
3 29
19 20
10 19
19 36
27 30
23 14
35 23
32 7
25 34
1 20
25 35
13 11
8 36
36 35
2 32
22 11
6 35
25 12
14 3
34 3
33 9
16 16
23 2
32 22
21 3
10 12
14 1
19 5
19 17
28 17
19 9
10 2
32 5
36 28
4 15
15 29
5 24
7 10
30 25
23 8
35 11
33 32
5 35
7 12
16 7
3 7
31 6
15 1
23 16
6 24
25 36
6 16
11 12
8 4
11 9
11 19
32 8
2 5
15 34
1 11
15 36
15 26
21 16
19 32
1 15
31 5
11 5
23 4
4 8
36 11
30 13
3 3
15 1
9 21
27 13
33 30
5 10
18 1
34 18
9 3
22 10
17 22
32 2
35 23
30 28
6 6
16 5
24 22
20 26
22 14
21 36
36 6
9 36
23 24
32 24
22 3
16 17
34 35
32 7
21 19
23 4
6 21
28 21
3 30
32 15
6 28
9 3
9 26
23 15
31 31
29 19
8 15
33 4
5 12
25 17
17 7
23 9
25 25
22 18
1 6
6 25
24 5
4 31
29 18
31 1
4 27
32 9
1 15
28 16
22 26
33 10
36 11
12 17
5 20
22 10
17 6
14 27
17 7
20 23
34 18
10 4
12 30
26 24
36 27
21 20
13 36
11 20
32 23
8 23
32 36
28 8
23 11
8 1
3 29
27 27
6 7
19 20
34 23
21 36
22 8
20 8
15 22
3 23
31 36
26 20
26 5
13 29
4 35
17 31
8 19
33 27
32 27
6 31
3 3
7 35
1 25
2 21
15 24
21 17
13 13
6 33
21 13
32 23
35 28
11 2
23 14
28 24
21 20
15 3
5 3
19 15
3 18
14 22
25 27
34 8
20 18
20 27
7 2
17 8
16 6
20 21
24 21
10 9
27 18
7 3
2 24
30 35
19 35
36 27
21 30
9 7
14 3
13 5
7 31
25 26
7 3
32 32
4 36
18 26
12 23
36 27
9 1
22 7
18 29
24 4
12 25
6 34
14 5
12 28
25 8
23 29
24 9
13 15
2 35
13 20
19 21
13 31
3 3
9 28
3 19
23 1
33 25
10 6
8 19
17 22
18 25
35 23
12 23
9 11
14 14
1 33
28 19
31 16
10 8
10 20
21 13
12 7
12 16
19 18
24 13
21 29
16 11
29 18
10 13
11 23
32 2
16 35
24 1
19 17
24 22
30 1
15 23
31 17
34 28
15 26
7 32
19 28
25 21
25 25
18 24
35 11
29 22
29 5
34 12
2 14
28 20
27 3
6 2
12 31
25 16
33 22
16 22
19 3
21 31
24 33
13 26
14 11
35 34
8 24
22 15
3 20
35 15
23 12
24 5
20 18
19 13
2 19
11 1
25 7
2 2
5 12
26 9
31 16
12 11
18 2
28 6
19 29
19 9
9 34
23 28
10 25
11 1
4 3
5 28
17 7
6 9
12 36
25 1
6 5
28 24
2 13
13 9
12 31
30 12
10 36
10 31
34 28
2 6
25 4
36 12
12 3
32 31
34 8
34 4
31 14
5 9
24 28
8 25
8 13
35 19
8 34
30 3
24 7
26 31
9 10
18 32
21 11
16 19
30 17
34 20
3 31
33 18
20 16
21 1
6 28
22 10
25 28
25 11
21 22
5 5
2 19
19 24
28 32
27 26
8 5
27 28
8 17
3 12
15 32
32 21
15 27
24 30
12 27
26 27
28 2
26 18
26 17
1 36
10 33
23 2
28 10
15 8
28 31
23 33
27 15
34 1
29 30
16 30
34 34
25 15
29 22
18 16
36 35
14 20
29 26
22 34
14 3
26 33
11 29
10 3
19 28
13 17
10 21
22 34
13 25
18 16
1 22
24 34
4 21
33 13
27 11
3 22
11 33
30 9
11 27
15 26
36 24
3 22
1 36
6 22
27 21
30 5
18 11
3 9
18 32
30 26
4 29
23 19
6 2
9 2
30 22
10 20
35 29
8 35
2 20
17 31
31 5
16 10
30 24
18 28
27 19
5 35
14 10
5 3
31 36
2 20
35 25
11 35
1 10
22 29
18 24
2 12
18 17
24 31
24 25
30 18
2 30
23 4
23 12
25 20
27 22
25 29
34 20
3 34
29 3
19 14
32 32
5 21
10 9
6 9
21 18
13 10
11 9
23 10
34 15
35 13
25 24
35 35
4 27
4 33
12 32
2 36
24 5
2 34
25 9
15 7
31 16
30 13
27 29
9 20
31 21
35 23
33 31
22 21
32 19
8 13
32 10
20 10
18 21
18 8
30 30
22 21
20 21
21 25
8 8
18 2
26 20
30 29
3 2
20 27
10 7
11 29
15 32
19 24
19 10
28 10
6 11
11 23
23 31
18 21
36 32
28 21
20 3
23 35
35 9
9 9
10 15
11 21
24 6
9 35
2 27
4 32
19 36
18 26
9 18
14 5
18 1
33 27
4 15
14 1
14 32
35 4
28 15
14 21
3 4
29 34
11 33
10 26
19 18
28 21
17 24
20 13
19 29
20 22
34 22
36 35
7 12
20 3
1 32
31 33
13 3
23 13
14 6
15 9
33 35
16 12
10 23
36 26
26 10
30 5
35 2
12 18
6 4
35 18
4 33
30 6
21 34
16 16
32 24
30 7
30 25
18 27
27 6
30 3
34 35
22 1
5 26
10 13
8 36
20 10
34 30
24 3
27 9
23 16
24 35
15 30
28 28
10 27
25 21
27 36
6 24
32 4
22 26
22 32
16 8
12 30
13 15
20 4
23 21
8 23
34 13
10 14
29 2
17 3
26 9
30 17
11 34
4 36
14 7
28 16
10 29
2 34
27 29
34 26
18 20
31 3
26 16
4 17
24 29
26 35
21 16
27 35
31 20
2 29
35 36
25 23
22 18
10 30
7 11
2 16
18 20
9 24
13 7
12 13
14 12
9 23
22 32
15 21
17 34
35 17
31 33
20 31
8 35
24 18
25 13
21 22
18 24
29 28
6 18
2 5
5 3
22 32
33 33
28 7
33 24
18 34
13 6
21 7
29 20
4 31
6 23
10 20
27 9
23 1
1 13
22 34
35 22
34 17
7 29
16 9
7 34
3 15
1 20
25 20
4 18
27 20
36 2
25 11
22 21
3 2
2 13
13 2
31 30
12 6
23 2
34 8
16 29
23 24
36 5
18 33
33 24
16 25
5 5
22 17
16 10
10 35
22 25
22 3
26 34
36 19
33 3
19 11
31 24
32 21
20 12
2 26
1 2
23 19
21 12
25 32
12 35
6 29
3 17
9 20
2 34
26 1
17 16
26 26
12 16
24 11
11 14
17 35
11 32
33 1
18 7
35 16
7 5
10 7
32 34
6 1
1 3
31 15
20 22
30 9
28 19
34 21
32 16
17 4
27 15
13 35
34 27
31 27
3 7
23 17
22 26
6 35
18 28
14 21
29 19
26 5
13 23
15 15
16 19
36 10
33 2
31 9
32 27
14 19
26 12
14 7
34 11
20 1
2 17
11 35
16 17
15 13
25 15
2 13
14 18
13 32
21 21
7 19
12 3
30 11
27 31
32 22
9 16
24 27
3 10
34 26
25 8
7 17
23 11
27 4
22 2
24 2
32 24
22 30
35 23
33 23
30 13
20 9
14 22
6 13
31 23
32 1
9 22
9 1
28 4
34 35
24 13
30 21
9 12
10 21
36 36
10 4
20 36
15 12
18 4
19 20
13 13
16 28
27 25
21 26
23 17
7 30
4 25
33 35
28 19
29 4
27 21
25 17
8 7
27 19
31 22
18 15
14 23
26 3
35 12
11 7
3 22
27 22
27 25
11 30
2 4
27 32
28 14
35 30
10 22
34 22
11 16
31 10
32 25
15 31
10 31
12 32
3 19
33 29
16 29
8 17
22 32
2 8
16 11
27 19
10 1
27 6
17 22
22 9
1 24
33 3
18 23
31 10
16 16
5 5
7 34
14 21
36 13
20 4
29 33
25 20
17 1
19 6
10 21
20 17
28 6
34 7
29 30
8 3
16 26
4 5
9 35
7 21
32 1
20 16
17 31
29 30
5 29
28 35
12 8
19 1
18 9
21 15
2 27
23 33
34 28
24 34
8 4
22 25
10 22
14 20
12 33
23 36
17 17
24 23
33 35
34 1
13 21
3 12
34 31
21 27
18 35
19 3
2 25
15 25
27 32
3 17
4 31
4 6
26 25
22 36
24 30
34 1
11 36
34 17
7 15
21 34
36 21
24 8
35 11
2 13
2 21
21 26
15 30
19 25
9 26
30 17
14 34
33 14
35 12
20 26
23 8
11 4
1 12
28 9
13 17
16 2
31 11
6 23
24 25
5 36
3 31
1 26
1 2
29 6
10 12
4 27
9 33
12 23
16 31
26 26
30 31
4 7
19 5
30 9
11 24
4 31
15 28
14 28
23 6
5 24
30 32
27 17
6 6
34 22
21 22
10 36
25 5
22 12
5 36
15 31
9 12
30 30
10 5
29 18
7 23
6 13
33 33
31 2
20 20
23 22
5 14
12 10
26 12
35 6
17 6
3 9
1 15
2 14
2 15
2 36
17 7
22 26
6 29
2 5
5 30
34 5
26 18
8 28
5 26
10 15
7 13
25 3
7 3
12 17
1 13
10 6
10 24
31 4
14 2
23 27
20 28
36 1
18 30
27 34
7 5
3 25
15 24
9 9
7 23
21 17
34 23
32 21
9 30
19 2
27 15
11 6
6 6
19 13
24 30
28 5
27 25
26 16
22 16
4 8
29 4
8 6
21 34
16 27
22 27
14 13
31 13
20 20
18 36
29 30
13 2
15 36
9 4
3 34
23 15
5 31
30 24
33 18
19 36
14 3
29 21
6 7
21 35
12 28
15 23
30 18
12 19
28 11
2 12
11 9
6 11
24 22
9 17
20 27
19 3
11 26
35 31
21 10
2 17
14 3
24 22
8 23
16 28
36 15
33 32
5 24
8 27
5 18
19 6
10 24
33 14
33 33
31 13
34 5
24 7
15 35
14 21
33 17
20 14
22 22
17 25
13 19
4 8
36 1
14 4
33 29
25 34
26 5
33 7
2 33
9 21
9 14
31 29
8 34
25 19
6 10
29 25
32 34
22 24
2 33
24 29
3 11
20 33
14 29
21 18
3 36
12 8
14 19
14 21
25 4
33 29
33 14
2 6
29 34
10 25
22 22
32 11
8 29
16 1
7 2
24 17
31 32
22 7
9 27
27 9
16 13
31 23
30 3
22 27
4 22
20 18
32 26
26 13
34 31
24 22
35 10
29 24
17 16
13 22
27 24
30 13
34 16
23 17
32 15
36 32
13 6
29 19
17 11
19 16
31 9
33 34
17 11
24 8
6 20
11 34
7 12
32 17
14 30
5 16
10 28
28 34
16 3
20 25
8 24
21 10
26 31
20 7
26 4
5 7
36 9
15 29
32 10
8 14
24 21
22 15
5 30
15 12
10 15
5 19
29 13
31 25
3 26
2 32
20 25
25 7
5 14
14 33
17 23
10 35
4 32
17 30
23 23
3 18
6 32
29 25
16 15
30 24
31 33
26 17
25 31
34 7
36 3
10 10
3 29
6 8
35 11
14 9
15 21
10 28
24 34
15 31
25 32
11 16
13 20
9 18
4 1
35 20
18 16
13 26
4 26
13 2
32 34
34 12
25 11
35 31
9 7
3 2
21 2
27 27
33 12
9 8
29 15
9 33
1 30
34 23
26 12
