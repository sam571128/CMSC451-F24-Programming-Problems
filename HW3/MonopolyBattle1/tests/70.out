471 473 530 649 471 338 649 312 825 312 312 518 649 338 416 312 854 334 338 268 268 471 268 236 312 268 365 268 664 649 473 268 312 236 365 268 